config/config.go

```diff
@@ -524,6 +525,29 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("stored_requests.http_events.amp_endpoint", "")
 	v.SetDefault("stored_requests.http_events.refresh_rate_seconds", 0)
 	v.SetDefault("stored_requests.http_events.timeout_ms", 0)
+	v.SetDefault("stored_video.filesystem.enabled", false)
```

Comment:
The use of the names StoredVideo and stored_video in this file and similarly named entries in other files below (e.g., videoFetcher) seem potentially confusing as they may be perceived as being related to stored videos rather than stored video requests (if not now, maybe someday in the future).  Perhaps it might be better to choose names that clearly indicate these all refer to stored video requests!?

Reply:
I can't imagine a scenario where we are storing actual videos on PBS. The closet we come is caching VAST XML on Prebid Cache. But if you can think of a name that is clearer and not an overlong monstrosity I am happy to update the config name.

Reply:
Maybe StoredVideoReq, stored_video_req, and videoReqFetcher?  An alternative would be comments in the code where each item is introduced.  If I'm the only one who cares, it's no bid deal if you don't change anything.

Reply:
I think I will go the comment route unless someone else comments thinking that this might be necessary.

Reply:
@hhhjort Not suggesting that this change is necessary in this PR but I think I agree with @josephveach and even though we don't see the potential of storing videos in future, it'd still improve readability if we rename to something more on the lines of `stored_video_req`. But as I said, doesn't have to be in this PR. Maybe we can leave this for another PR. Refactoring and adding features in the same PR can be a little too much most of the times.

Reply:
Is it really "refactoring" given the code in question is being added here for the first time?  :-)

Reply:
I will leave potentially renaming the struct fields for another PR. But the fields in the config file would be an API change, so should get them right on launch. So the above line will be modified along with its friends.